src/Components/Header/index.js

```diff
@@ -11,7 +11,7 @@ const Header = ({openComponent}) => {
             <Button 
                 h="35px"
```

Comment:
I think "h" and "w" is not a good names for props, I understand that they mean width and height but you need to change these names of props